### DISCUSSION TITLE
formtab组件第二次加载后，无法切换tab

### DISCUSSION THREAD

**Discussion by feixia5320** [alibaba/formily#2165] [Q&A 1.x] [answered]
formtab组件第二次加载后，无法切换tab
比如我在form组件前几个toggle切换显隐功能，第二次显示后，无法切换tab了
<div>
      <button onClick={btnClick}>toggle</button>
      {show && (
        <FormProvider form={form}>
          <SchemaField>
            <SchemaField.Void
              type="void"
              x-component="FormTab"
              x-component-props={{ formTab }}
            >
              <SchemaField.Void
                type="void"
                name="tab1"
                x-component="FormTab.TabPane"
                x-component-props={{ tab: "A1", key: "1" }}
              >

demo如下
https://codesandbox.io/s/nervous-visvesvaraya-7mnmc?file=/App.tsx

**Comment by janryWang** [comment, marked as answer]
升级rc.8

**Reply by feixia5320** [reply]
升级后解决了，多谢
